libs/cardano-ledger-pretty/cardano-ledger-pretty.cabal

```
@@ -42,7 +42,7 @@ library
         cardano-ledger-conway >=1.3,
         cardano-ledger-core ^>=1.3,
         cardano-ledger-mary >=1.0,
-        cardano-ledger-shelley ^>=1.3,
+        cardano-ledger-shelley ^>=1.4,
```

Also needs a version bump to `1.2.0.1`
```suggestion
        cardano-ledger-shelley >=1.3 && <1.5,
```